Releases.md

```diff
@@ -6,6 +6,16 @@ https://github.com/denoland/deno/releases
 We also have one-line install commands at:
 https://github.com/denoland/deno_install
 
+### 1.21.3 / 2022.05.12
+
+- feat(cli): add deno version to manual links (#14505)
```

Comment:
Maybe let's rewrite it to `fix(cli)`?

Reply:
Sure.
Do you want to rewrite the commit message as well, or just the releases.md entry?